osmnx/geometries.py

```diff
@@ -632,43 +632,40 @@ def _is_closed_way_a_polygon(element, polygon_features=_polygon_features):
             # identify common keys in element's tags and polygon_features dict
             intersecting_keys = element_tags.keys() & polygon_features.keys()
 
-            # if common keys are found
```

Comment:
@AtelierLibre the changes in this function would particularly benefit from your eye to verify if the logic remains consistent with your original design.

Reply:
@gboeing unless I'm missing something, doesn't this change  just remove the `if len(intersecting_keys) > 0:` check (which I can see is probably redundant) and then outdent the rest of the code?